pyccel/codegen/printing/fcode.py

```diff
@@ -1646,6 +1646,26 @@ def _do_range(target, iterable, prolog, epilog):
 
     # .....................................................
     #                   OpenMP statements
+    # .....................................................
+    def _print_OMP_Parallel_Construct(self, expr):
+        ParallelConstruct   = str(expr.txt)
+        ompexpr = '!$omp {}\n'.format(ParallelConstruct)
```

Comment:
`ParallelConstruct` is showing up in yellow on my computer. I suspect that the use of `CamelCase` encourages github to consider that it is a class. It would therefore maybe better to use `parallel_construct` (see https://en.wikipedia.org/wiki/Naming_convention_(programming)#Python_and_Ruby)